Gemfile

```diff
@@ -5,6 +5,7 @@ gemspec
 
 group :development, :test do
   gem 'coveralls', require: false
+  gem 'easy_translate'
```

Comment:
Can you add documentation on what this is for and how you use it?

Reply:
Does the commit message for 579eb89 satisfy this?

Reply:
i.e. there are no changes, except that this dependency used to be added by `i18n-tools`, and now it is not

Reply:
Yes, sorry, I missed that because of the folding.